.gitignore

```diff
@@ -1,34 +1,13 @@
 *.[oa]
 *.deps
-src/cdxxx.c
-src/debtab.c
 src/dmd
-src/dmd_frontend
-src/ddmd
-src/dmd.conf
-src/elxxx.c
-src/fltables.c
-src/id.c
-src/id.h
-src/id.d
-src/idgen
-src/optab.c
-src/optabgen
-src/tytab.c
 test/test_results
+test/trace.def
+test/trace.log
+generated/
 *.obj
-src/idgen.exe
 src/dmd.exe
-src/dmd_frontend.exe
-src/optabgen.exe
-src/verstr.h
 *.map
-.DS_Store
```

Comment:
I think this one should stay on this list: https://en.wikipedia.org/wiki/.DS_Store